assets/python/ml/pyml:model:multilayer_perceptron:sklearn.j2.py

```diff
@@ -25,31 +28,54 @@
 with settings.context as context:
     # Train
     if settings.is_workflow_running_to_train:
-        # Restore data
-        descriptors = context.load("descriptors")
-        target = context.load("target")
+        # Restore the data
+        train_target = context.load("train_target")
+        train_descriptors = context.load("train_descriptors")
+        test_target = context.load("test_target")
+        test_descriptors = context.load("test_descriptors")
 
-        # Transform targets from shape (100,1) to shape (100,); required by sklearn's MLP Regressor
-        target = target.ravel()
+        # Flatten the targets
+        train_target = train_target.flatten()
+        test_target = test_target.flatten()
 
         # Initialize the NN model
         model = sklearn.neural_network.MLPRegressor(hidden_layer_sizes=(100,),
                                                     activation="relu",
                                                     solver="adam",
-                                                    learning_rate="adaptive",
-                                                    max_iter=500)
+                                                    learning_rate_init=0.001,
+                                                    max_iter=200,
+                                                    shuffle=True,
+                                                    random_state=True,
+                                                    tol=1e-4,
+                                                    verbose=False,
+                                                    early_stopping=False,
+                                                    validation_fraction=0.1,
+                                                    beta_1=0.9,
+                                                    beta_2=0.999,
+                                                    epsilon=1e-8,
+                                                    n_iter_no_change=10)
```

Comment:
Lots of parameters instead of just a few before ... Are you sure about all of them necessary?

Reply:
The thought here was to expose all the knobs involved, but they can always read the sklearn documentation. I've slimmed this down to just a few key (or common) parameters: 
- hidden layer sizes
- activation
- solver
- max_iter
- early_stopping
- validation_fraction